packages/std/src/math/uint256.rs

```diff
@@ -102,6 +102,18 @@ impl Uint256 {
         Uint256(U256(words))
     }
 
+    /// A conversion from `u128` that, unlike the one provided by the `From` trait,
+    /// can be used in a `const` context.
+    pub const fn from_u128(num: u128) -> Self {
+        let bytes = num.to_le_bytes();
+
+        Self::from_le_bytes([
+            bytes[0], bytes[1], bytes[2], bytes[3], bytes[4], bytes[5], bytes[6], bytes[7],
+            bytes[8], bytes[9], bytes[10], bytes[11], bytes[12], bytes[13], bytes[14], bytes[15],
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        ])
+    }
```

Comment:
Can we dedup the implementation with `from_uint128` (use one from the other)?

Reply:
Of course, good point. I jumped to the test and forgot to optimize implementation.

I did it in reverse though, so that `from_uint128` calls `from_u128` (because otherwise a new `Uint128` would have to be created).

Reply:
Sure, looks good